reporters/html_reporter.py

```diff
@@ -0,0 +1,25 @@
+from reporters.plain_reporter import PlainReporter
+from jinja2 import Environment
+
+
+class HTMLReporter(PlainReporter):
+    def __init__(self):
+        super().__init__()
+
+    # Override this method
+    def print_message_ids(self):
+        # Sort the messages.
+        self.sort_messages()
+
+        #template = Environment(keep_trailing_newline=True, autoescape=False).from_string(HTML)
+        template = Environment().get_template('template.txt')
+        f = open('output.html', 'w')
```

Comment:
Use `with open(...) as f: ...` (you may need to look up the purpose of this syntax)